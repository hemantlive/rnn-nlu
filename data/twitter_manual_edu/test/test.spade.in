<s> It was Apple Store in ManchesterArndale, I had silver phone and was given a rose gold. Now in USA and wasn't sure how I can get it swapped? </s>
<s> @AppleSupport why is voice searching not working on my Apple TV? It just hangs then crashes </s>
<s> All my brother and sister wanted was a street wise 2 and an avocado twista. what they got was unacceptable and unhygienic. </s>
<s> @kfc fast food at KFC Glastonbury? What a joke , waited 10 minutes after ordering to be told at the window it's a 20 min wait </s>
<s> @Citakki Good to know. Do you see any error messages in Nike+ Connect when trying to sync? </s>
<s> @markirv11 Sorry to hear that. Give these steps a shot and let us know how it goes: http://gonike.me/6016dxJ6 . Any luck? </s>
<s> Don’t sync just yet, we’re taking some time to work on Nike+. You down to hit the road?  </s>
<s> Let's look into what's going on with this. Could you explain the steps you're taking to copy and which app you are using? </s>
