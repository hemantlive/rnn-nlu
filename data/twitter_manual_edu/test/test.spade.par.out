(Root (span 1 6)
  ( Nucleus (span 1 4) (rel2par Joint)
    ( Nucleus (leaf 1) (rel2par Joint)
(text _!It was Apple Store in ManchesterArndale ,_!) )
    ( Nucleus (span 2 4) (rel2par Joint)
      ( Nucleus (leaf 2) (rel2par Joint)
(text _!I had silver phone_!) )
      ( Nucleus (span 3 4) (rel2par Joint)
        ( Nucleus (leaf 3) (rel2par span)
(text _!and was given a rose_!) )
        ( Satellite (leaf 4) (rel2par Elaboration)
(text _!gold. Now in USA_!) )
       )
     )
   )
  ( Nucleus (span 5 6) (rel2par Elaboration)
    ( Satellite (leaf 5) (rel2par Attribution)
(text _!and was n't sure_!) )
    ( Nucleus (leaf 6) (rel2par span)
(text _!how I can get it swapped ?_!) )
   )
)
(Root (span 1 3)
  ( Nucleus (leaf 1) (rel2par span)
(text _!@ AppleSupport_!) )
  ( Satellite (span 2 3) (rel2par Elaboration)
    ( Satellite (leaf 2) (rel2par Attribution)
(text _!why is voice searching not working on my Apple TV ? It just hangs_!) )
    ( Nucleus (leaf 3) (rel2par span)
(text _!then crashes_!) )
   )
)
(Root (span 1 4)
  ( Nucleus (leaf 1) (rel2par Same-Unit)
(text _!All my brother and sister_!) )
  ( Nucleus (span 2 4) (rel2par Same-Unit)
    ( Nucleus (leaf 2) (rel2par Joint)
(text _!wanted was a street wise 2_!) )
    ( Nucleus (span 3 4) (rel2par Joint)
      ( Satellite (leaf 3) (rel2par Attribution)
(text _!and an avocado twista._!) )
      ( Nucleus (leaf 4) (rel2par span)
(text _!what they got was unacceptable and unhygienic ._!) )
     )
   )
)
(Root (span 1 3)
  ( Nucleus (leaf 1) (rel2par span)
(text _!@ kfc fast food at KFC Glastonbury ? What a joke ,_!) )
  ( Satellite (span 2 3) (rel2par Elaboration)
    ( Satellite (leaf 2) (rel2par Attribution)
(text _!waited 10 minutes after ordering to be told at the window_!) )
    ( Nucleus (leaf 3) (rel2par span)
(text _!it 's a 20 min wait_!) )
   )
)
(Root (span 1 4)
  ( Satellite (leaf 1) (rel2par Attribution)
(text _!@ Citakki Good to know._!) )
  ( Nucleus (span 2 4) (rel2par span)
    ( Nucleus (leaf 2) (rel2par span)
(text _!Do you see any error_!) )
    ( Satellite (span 3 4) (rel2par Elaboration)
      ( Nucleus (leaf 3) (rel2par span)
(text _!messages in Nike+ Connect_!) )
      ( Satellite (leaf 4) (rel2par Background)
(text _!when trying to sync ?_!) )
     )
   )
)
(Root (span 1 4)
  ( Nucleus (span 1 3) (rel2par span)
    ( Nucleus (span 1 2) (rel2par span)
      ( Nucleus (leaf 1) (rel2par Joint)
(text _!@ markirv11 Sorry to hear that. Give these steps a shot_!) )
      ( Nucleus (leaf 2) (rel2par Joint)
(text _!and let us know_!) )
     )
    ( Satellite (leaf 3) (rel2par Joint)
(text _!how it goes : http :_!) )
   )
  ( Satellite (leaf 4) (rel2par Joint)
(text _!//gonike.me/6016dxJ6 . Any luck ?_!) )
)
(Root (span 1 3)
  ( Nucleus (leaf 1) (rel2par span)
(text _!Don’t sync just yet , we’re taking some time_!) )
  ( Satellite (span 2 3) (rel2par Elaboration)
    ( Nucleus (leaf 2) (rel2par span)
(text _!to work on Nike+._!) )
    ( Satellite (leaf 3) (rel2par Enablement)
(text _!You down to hit the road ?_!) )
   )
)
(Root (span 1 5)
  ( Nucleus (leaf 1) (rel2par Temporal)
(text _!Let 's look into what 's going on with this._!) )
  ( Nucleus (span 2 5) (rel2par Temporal)
    ( Nucleus (leaf 2) (rel2par span)
(text _!Could you explain the steps_!) )
    ( Satellite (span 3 5) (rel2par Elaboration)
      ( Nucleus (leaf 3) (rel2par span)
(text _!you 're taking to copy_!) )
      ( Satellite (span 4 5) (rel2par Elaboration)
        ( Satellite (leaf 4) (rel2par Attribution)
(text _!and which app_!) )
        ( Nucleus (leaf 5) (rel2par span)
(text _!you are using ?_!) )
       )
     )
   )
)
